bin/fiberassign

```diff
@@ -42,9 +42,17 @@ parser.add_argument("--nstarpetal", type=int, help="number of standard stars per
 parser.add_argument("--nskypetal", type=int, help="number of sky fibers per petal (default=40)", default=40)
 
 parser.add_argument("--nocleanup", dest='cleanup', default=True, action='store_false')
+parser.add_argument("--version",help='Print C code version and exit',action='store_true')
```

Comment:
You should use `action='version'`, then you should not need to remove the required arguments.  See https://docs.python.org/3/library/argparse.html#action.  If necessary, get the version from the C++ code beforehand to pass set in `parser.add_argument()`.

Reply:
Nice, thanks. I've made that change and restored the `required=True` to the other options.